sources/org.osbuild.git

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python3
+
+import json
+import os
+import sys
+import subprocess
+import tempfile
+
+
+STAGE_DESC = "Fetch the contents of a git commit"
```

Comment:
How does this relate to this PR?  Maybe it's aiming to be analogous to what we do with the `overlay.d/` directory in e.g. https://github.com/coreos/fedora-coreos-config ?

Reply:
I indeed use it [right now](https://github.com/gicmo/ostree-osbuild-demo/pull/23/files#diff-1271af4bdd6ca3ac7f2361df048bdd8cR1077) to pull in the `40ignition-ostree` module from `fedora-coreos-config`, since that is not available as rpm but needed to glue ostree and ignition together (which you of course know ;D)